README.md

```diff
@@ -1,49 +1,95 @@
-django.nV
```

Comment:
Why did you modify the README.md of the nVisium repo instead of just making a vulnerability report file?